### DISCUSSION TITLE
Matter iot with micropython?

### DISCUSSION THREAD

**Discussion by plocher** [micropython/micropython#10633] [General]
Connectedhome's Matter tooling includes an embedded ipython scripting repl for its Linux controller...

Is anyone thinking about the other end, creating Matter aware nodes in micropython?

-John
Www.SPCoast.com

**Comment by andypiper** [comment]
I'm curious about it, and attended an ARM+Sparkfun webinar about Matter the other week - I asked if they have plans for MicroPython support for their boards, but didn't get a clear answer. I'm guessing it will be a case of waiting for some of the dev boards to become more available, and then drivers for the radios etc to support it.
